sonar-project.properties

```diff
@@ -2,8 +2,6 @@ sonar.projectKey=eslint-plugin-sonarjs
 sonar.projectName=eslint-plugin-sonarjs
 sonar.projectDescription=SonarJS rules for ESLint
 
-# current version in development
-# should be bigger than the one in package.json
 sonar.projectVersion=0.14.0
```

Comment:
As I didn't change this version we gonna have wrong new code period in SC until next release